ktor-server/ktor-server-host-common/jvm/src/io/ktor/server/engine/BaseApplicationEngine.kt

```diff
@@ -35,8 +36,10 @@ public abstract class BaseApplicationEngine(
             it.sendPipeline.installDefaultTransformations()
         }
         environment.monitor.subscribe(ApplicationStarted) {
+            val initializeFinishedAt = System.nanoTime()
             environment.connectors.forEach {
                 environment.log.info("Responding at ${it.type.name.toLowerCase()}://${it.host}:${it.port}")
+                environment.log.info("Startup time: ${(initializeFinishedAt - initializeStartedAt) / 1_000_000} ms")
```

Comment:
Sorry, didn't noticed that but why do we print the same message for evey connector?